ClassComments-Via-Class

MarcusDenker opened the pull request:
This PR changes all cliens to as the class directly for comments, not the #organizer

This prepares to move the classCommentSourcePointer to ClassDescription

This turns for now off the automatic class template, this should be done in the UI, not in the model

MarcusDenker committed:
This PR changes all cliens to as the class directly for comments, not the #organizer

This prepares to move the classCommentSourcePointer to ClassDescription

This turns for now off the automatic class template, this should be done in the UI, not in the model

privat commented on src/Kernel/ClassDescription.class.st:
```diff
@@ -334,8 +333,7 @@ ClassDescription >> classesThatImplementAllOf: selectorSet [
 { #category : #'accessing - comment' }
 ClassDescription >> comment [
-    "Answer the receiver's comment. (If missing, supply a template) "
-    ^self instanceSide organization classComment ifEmpty: [ self classCommentBlank ]
```
yes!

jecisc reviewed (approved):
I like to see all those #organization vanish :)

Do you plan to move the empty class template in the UI soon? (just wondering)

privat commented:
I like the removal of the template, in a model POV I think that "no comment" is important information to preserve.
The template should be used when developers want to comment the class: i.e. not on default presentation, but when entering in edition mode.

MarcusDenker committed:
fix tests
- testNoUnusedInstanceVariablesLeft
- I removed testFromClassWithoutCommentWhenPatternSatisfiesCommentTemplate
